Sources/SwiftDocC/SwiftDocC.docc/Resources/LinkableEntities.json

```diff
@@ -67,7 +67,7 @@
                     "usr": {
                         "type": "string"
                     },
-                    "declarationFragments": {
+                    "fragments": {
```

Comment:
I think this has been incorrect for some time. While the Swift property is named `declarationFragments`, we encode and decode it as `fragments`.